src/features/create-dao/steps/GovernanceForm.test.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+
+import { render, waitFor, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { ZUIProvider } from '@zero-tech/zui/ZUIProvider';
+import { GovernanceForm, GovernanceFormProps } from './';
+import { CreateDAOFormContext } from '../';
+
+let onSubmit = jest.fn();
+
+const DEFAULT_PROPS: GovernanceFormProps = {
+	onClose: jest.fn(),
+};
+
+const DEFAULT_PROVIDER_VALUES = {
+	stepId: 'details',
+	title: 'Create DAO',
+	details: {
+		mediaType: undefined,
+		previewUrl: '',
+		name: '',
+		znaAddress: '',
+		description: '',
+	},
+	governance: {
+		votingProcess: 'absolute',
+		votingPeriod: '',
+		votingSystem: '',
+		daoTokenAddress: '',
+		votingThreshold: '',
+	},
+	treasury: {
+		gnosisSafe: '',
+	},
+	onStepUpdate: jest.fn(),
+	onTitleUpdate: jest.fn(),
+	onDetailsChange: jest.fn(),
+	onDetailsSubmit: jest.fn(),
+	onGovernanceSubmit: onSubmit,
+	onTreasurySubmit: jest.fn(),
+	onLaunchSubmit: jest.fn(),
+};
+
+// We need to mock ResizeObserver to test the dropdown menu's.
+// See https://github.com/radix-ui/primitives/issues/420#issuecomment-1125837782.
+(global as any).ResizeObserver = class ResizeObserver {
+	constructor(cb: any) {
+		(this as any).cb = cb;
+	}
+
+	observe() {
+		(this as any).cb([{ borderBoxSize: { inlineSize: 0, blockSize: 0 } }]);
+	}
+
+	unobserve() {}
+};
```

Comment:
Can we move this to the global jest config, so we don't have to add it in each test suite?